lib/ci-stack.ts

```diff
@@ -6,21 +6,58 @@
  * compatible open source license.
  */
 
-import { Construct, Stack, StackProps } from '@aws-cdk/core';
+import { Vpc } from '@aws-cdk/aws-ec2';
+import { Secret } from '@aws-cdk/aws-secretsmanager';
+import {
+  CfnParameter,
+  Construct, Fn, Stack, StackProps,
+} from '@aws-cdk/core';
+import { ListenerCertificate } from '@aws-cdk/aws-elasticloadbalancingv2';
+import { CIConfigStack } from './ci-config-stack';
+import { JenkinsMainNode } from './compute/jenkins-main-node';
+import { JenkinsMonitoring } from './monitoring/ci-alarms';
+import { JenkinsExternalLoadBalancer } from './network/ci-external-load-balancer';
+import { JenkinsSecurityGroups } from './security/ci-security-groups';
 
 export class CIStack extends Stack {
   // eslint-disable-next-line no-useless-constructor
   constructor(scope: Construct, id: string, props?: StackProps) {
     super(scope, id, props);
 
-    // Network / VPC
+    const vpc = new Vpc(this, 'JenkinsVPC');
 
-    // Security Groups
+    const useSslParameter = new CfnParameter(this, 'useSsl', {
+      description: 'If the jenkins instance should be access via SSL',
+      allowedValues: ['true', 'false'],
```

Comment:
Can we use `y` or `n` instead? It would be lesser syllables to type.

Reply:
It looks like `npm run cdk deploy CI-Dev -- --parameters useSsl=false` in practice which I think is more readable.

Reply:
Would it not ask as a y/n question when deploying, something like - 
`Are you sure you want to delete: JenkinsStack (y/n)? y`?